generators/client-2/templates/webpack/_webpack.common.js

```diff
@@ -95,7 +95,21 @@ module.exports = function (options) {
             ]),
             new webpack.ProvidePlugin({
                 $: "jquery",
-                jQuery: "jquery"
+                jQuery: "jquery",
+                "window.jQuery": "jquery",
+                'Tether': 'tether',
+                'window.Tether': 'tether',
+                Tooltip: "exports?Tooltip!bootstrap/js/dist/tooltip",
```

Comment:
we dont need any bootstrap JS stuff. We use only css from here. For js we use ng-bootstrap